as_perimenoun_oi_gynaikes.ass

```diff
@@ -1066,11 +1067,11 @@ Dialogue: 0,1:16:24.56,1:16:27.54,Default,,0,0,0,,My wife didn't do anything out
 Dialogue: 0,1:16:27.54,1:16:29.79,Default,,0,0,0,,And she calls me two days ago and she was acting all nice...
 Dialogue: 0,1:16:29.79,1:16:31.26,Default,,0,0,0,,"Panos, when are you coming?"
 Dialogue: 0,1:16:31.26,1:16:33.01,Default,,0,0,0,,"We've been alone for a month."
-Dialogue: 0,1:16:33.01,1:16:35.51,Default,,0,0,0,,Like she had a clue.
+Dialogue: 0,1:16:33.01,1:16:35.51,Default,,0,0,0,,Like they had a clue.
 Dialogue: 0,1:16:35.51,1:16:36.94,Default,,0,0,0,,Doesn't matter.
-Dialogue: 0,1:16:36.94,1:16:38.94,Default,,0,0,0,,Are they having a good time now?
+Dialogue: 0,1:16:36.94,1:16:38.94,Default,,0,0,0,,Aren't they triggered now?
 Dialogue: 0,1:16:38.94,1:16:41.44,Default,,0,0,0,,Once we get there, are they going to annoy our balls off?
-Dialogue: 0,1:16:41.44,1:16:42.84,Default,,0,0,0,,We just don't get it, Panos.
+Dialogue: 0,1:16:41.44,1:16:42.84,Default,,0,0,0,,We don't give a fuck, Panos.

```

Comment:
Why make a "don't give a fuck" out of  a line which in effect says "we don't know".

Reply:
"Δεν καταλαβαίνω" in greek is not always equivalent to "not understand". Here it means that we don't care.

For example "δεν καταλαβαίνω Χριστό/μια, θα τον δείρω" 

https://youtu.be/ZGUJa7ZF2Y4

Reply:
OK, but "We don't give a fuck" means "we don't care". We need to find something more descriptive here like "We will get through it, Panos." or "We will manage, Panos."

Reply:
Yeah 'We will get through it, Panos.' is a better translation.